std/container/rbtree.d

```diff
@@ -1842,6 +1850,19 @@ pure unittest
     assert(array(rt4[]) == ["hello"]);
 }
 
+unittest {
+    import std.conv : to;
+
+    auto rt1 = redBlackTree!string();
+    assert(rt1.to!string == "RedBlackTree([])");
```

Comment:
They're intended. They explicitly show that the tree is empty, where as `RedBlackTree()` could be interpreted as just saying "this is a struct of type RedBlackTree".